dpgen/generator/run.py

```diff
@@ -41,6 +41,11 @@
 from dpgen.generator.lib.utils import log_task
 from dpgen.generator.lib.utils import symlink_user_forward_files
 from dpgen.generator.lib.lammps import make_lammps_input, get_dumped_forces
+from dpgen.generator.lib.calypso import make_run_opt_script,make_check_outcar_script
+from dpgen.generator.lib.calypso import _make_model_devi_native_calypso,write_model_devi_out,_make_model_devi_buffet
+from dpgen.generator.lib.model_devi_calypso import gen_structures,analysis
+from dpgen.generator.lib.write_modd import write_modd
```

Comment:
As @amcadmus suggests, such functions should be named `write_model_devi_calypso`, etc.